pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -883,34 +878,17 @@ def process_circuits(
                     for name, count in Counter(bit.reg_name for bit in c0.bits).items():
                         for i in range(count):
                             results_selection.append((name, i))
-                    try:
```

Comment:
Should we still do a try and catch for possible errors and give more specific details that the conversion failed and they could try the qasm way if they want to?

Reply:
Currently the only message we print on error is "try installing pytket-qir", and that shouldn't be necessary now. Better error messages could be added in another PR, though the error raised by `pytket_to_qir()` should contain at least some useful information.

Reply:
Yes, but that error will not contain any hints pytket-quantinuum specific. Happy to do this in a separate PR.